### ISSUE TITLE
Have tagged versions available in Maven Central

### ISSUE THREAD

**Issue by kiview** [testcontainers/testcontainers-spock#22]
Although Jitpack is a good solution, a mature library should provide their artifacts in Maven Central as well.

**Comment by iNikem** [comment]
It may be much easier to publish to Bintray: https://github.com/bintray/gradle-bintray-plugin

**Comment by kiview** [comment]
That's true, but I was planning on having it in the same Maven Central group as testcontainers-java. Also I think having it in Bintray isn't that much of an improvement over jitpack.io, since it isn't a Maven/Gradle default?

**Comment by iNikem** [comment]
What do you mean "default"? Gradle has no default central repository. It is true, that the majority of examples use `mavenCentral()`, but it is similarly easy to use `jcenter()` in gradle script. It is HUGE improvement over jitpack, as it gives clear versioning to all users.

Besides, although I have not done this, but people say it is MUCH easier to publish to Maven Central via Bintray.

**Comment by kiview** [comment]
`mavenCentral()` is still the default for Maven, that's the reason. But I definitely see your point. Still like to release it in a similar fashion to testcontainers-java.

**Comment by iNikem** [comment]
This may be of help: https://bintray.com/docs/usermanual/uploads/uploads_syncingwiththirdpartyplatforms.html

**Comment by kiview** [comment]
Btw. would it help to have tagged version in Github, so you can depend on tagged versions using Jitpack?

**Comment by kiview** [comment]
I've added a tagged release for the current version:
`testCompile 'com.github.testcontainers:testcontainers-spock:1.0.0-tc1.6.0'`

Since we'll likely move this repo into the testcontainers-java repo (see https://github.com/testcontainers/testcontainers-java/issues/564), I won't do further work on this issue in this project itself.
